presentation/common/package.json

```diff
@@ -18,14 +18,17 @@
     "name": "Bentley Systems, Inc.",
     "url": "http://www.bentley.com"
   },
-  "main": "lib/presentation-common.js",
-  "typings": "lib/presentation-common",
+  "main": "lib/cjs/presentation-common.js",
+  "module": "lib/esm/presentation-common.js",
+  "typings": "lib/cjs/presentation-common",
   "scripts": {
```

Comment:
> I don't think `assets` need to be copied to `lib/esm` - they're only used on backends, so copying to `lib/cjs` should be suffient.

They still need to be copied to `public`. The assets in `common` package are kind of special, because they may be used by either backend or frontend, and they look at different places. Backends look at `assets` and frontends look at `public`. So `cjs` builds needs to have them copied to both `assets` and `public`, and `esm` builds need to have them copied only to `public`.

Reply:
Re-added copying to esm, closing this comment. Will discuss localization comment in another open one below.